tensorboard/webapp/metrics/BUILD

```diff
@@ -47,6 +47,7 @@ tf_ts_library(
     visibility = ["//tensorboard/webapp/metrics:__subpackages__"],
     deps = [
         "//tensorboard/webapp/widgets/histogram:types",
+        "//tensorboard/webapp/widgets/linked_time_types",
```

Comment:
This is ok but I can't help think "A linked_time_type is not a widget".

Here is another idea but I ultimately leave it up to you to decide if you think it is worthwhile/better:
* Move linked_time_types.ts into linked_time_fob.
* Optionally, in addition: Rename linked_time_fob folder to linked_time.
The end result is sort of how widgets/histogram is organized.

Reply:
That's a fair question I thought about it too.
For now I'm a bit reluctant to remove "fob" from "widget/linked_time_fob" since the key functionalities of the widget is to control the interactions/movements of those fobs.
In widget/linked_time_fob  there might be some types used only in linked_time_fob (for example, Fob enum).

Now my thoughts are 
* widget/linked_time_fob/internal_types: types used only in linked_time_fob
* widget/linked_time_fob/types: those will be reused in other widgets and metrics/

What do you think?

Reply:
I implemented the linked_time_fob/types in this PR9https://github.com/tensorflow/tensorboard/pull/5605/files). The Fob enum could be moved to an internal_types file.

Reply:
SGTM. You could additionally try enforcing the internalness of internal_types by putting it in its own BUILD rule and reducing the visibility of the BUILD rule.